app/services/album.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios'),
+  config = require('../../config'),
+  base_uri = config.common.albumsApi.url,
+  logger = require('../logger');
+
+exports.getAlbum = params =>
+  axios
+    .get(`${base_uri}/albums?id=${params.id}`)
+    .then(response => response.data[0])
+    .catch(error => {
+      logger.error(error);
```

Comment:
Here you are catching the error, so it will be logged but function `getAlbum` will return undefined on exceptions. 

You should throw an error here, I recommend you to do:

```javascript
logger.error(error);
throw new Error('Cannot fetch album from external api');
```